consensus/polybft/sc_integration_test.go

```diff
@@ -201,22 +208,22 @@ func TestIntegration_CommitEpoch(t *testing.T) {
 			result.GasUsed,
 		)
 
+		// Normally injecting balance to the reward wallet is handled by a higher order method in the executor.go
```

Comment:
You've misunderstood it here. The systemCaller balance is increased when he is executing a transaction that needs a transfer of Hydra. The reward wallet is injected by the special fund transaction you've implemented